client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -480,8 +481,8 @@ public void close() throws SegmentSealedException {
             return;
         }
         log.debug("Closing writer: {}", writerId);
-        // Wait until all the inflight events are written
-        flush();
+        // Wait until all the in-flight events are written
+        clearInFlight(new CloseAppend(writerId, segmentName));
```

Comment:
The name 'clear' in this context is misleading.

Reply:
ok, I have changed to flushInFlight